src/reference/00-Getting-Started/05-Basic-Def.md

```diff
@@ -138,8 +97,8 @@ the value `"hello"`.
 If you use the wrong value type, the build definition will not compile:
```

Comment:
Above the text says

> In this case, the returned `Setting[String]` is a transformation to add or replace the `name` key in sbt's map, giving it the value `"hello"`.

But we just said the map is immutable. Can we maybe not claim it's immutable?